pkg/types/defaults.go

```diff
@@ -8,7 +8,30 @@ const (
 	DefaultMTU         int = 1500
 )
 
-var NcpSections = []string{"DEFAULT", "ha", "k8s", "coe", "nsx_v3", "vc"}
-var AgentSections = []string{"DEFAULT", "k8s", "coe", "nsx_node_agent", "nsx_kube_proxy"}
-var OperatorSections = []string{"DEFAULT", "ha", "k8s", "coe", "nsx_v3", "vc", "nsx_node_agent", "nsx_kube_proxy"}
-var BootstrapOptions = []string{"DEFAULT", "nsx_node_agent"}
+var (
+	NcpSections      = []string{"DEFAULT", "ha", "k8s", "coe", "nsx_v3", "vc"}
+	AgentSections    = []string{"DEFAULT", "k8s", "coe", "nsx_node_agent", "nsx_kube_proxy"}
+	OperatorSections = []string{"DEFAULT", "ha", "k8s", "coe", "nsx_v3", "vc", "nsx_node_agent", "nsx_kube_proxy"}
+	BootstrapOptions = []string{"DEFAULT", "nsx_node_agent"}
```

Comment:
It seems that we don't need section `vc` anymore?

Reply:
yes, that is true, we can remove vc section